src/core/dep-graph.ts

```diff
@@ -317,30 +326,56 @@ class DepGraphImpl implements types.DepGraphInternal {
   private pathsFromNodeToRoot(
```

Comment:
This is a private method that I verified no one is using this - https://github.com/search?q=org%3Asnyk+pathsFromNodeToRoot&type=code